### ISSUE TITLE
Deprecate TGARCH to TARCH or GJRGARCH?

### ISSUE THREAD

**Issue by s-broda** [s-broda/ARCHModels.jl#106]

**Comment by waynelapierre** [comment]
GJR-GARCH would be a great feature to have. I use the R packge rugarch to estimate GJR-GARCH and it is super slow.

**Comment by s-broda** [comment]
It's definitely within scope, I just need to find the time to implement it. PRs are welcome :)

**Comment by ASaragga** [comment]
First, I would like to thank you very much for this excellent package. I would also like to second the plan to deprecate the name TGARCH in favour of GJRGARCH (or similar).

In ARCHModels, the "TGARCH" implementation corresponds to the GJR-GARCH model introduced by  Glosten, Jagannathan, and Runkle (1993,) as correctly cited in the documentation. This is because, I believe, the threshold GARCH (TGARCH) terminology is sometimes used interchangeably with the GJR-GARCH model, even though technically TGARCH can also refer to a slightly different specification, due to Zakoian (1994). So as it is now this terminology can be a  bit confusing.

**Comment by s-broda** [comment]
I agree in principle, just lack the time for code churn at the moment. PRs are welcome :)
